File: app/src/main/java/com/possible/demo/Follower.java
package com.possible.demo;

import com.google.gson.annotations.SerializedName;


public class Follower {

    @SerializedName("login")
    public String login;

    @SerializedName("id")
    public String id;

    @SerializedName("avatar_url")
    public String avatarUrl;
}

File: app/src/main/java/com/possible/demo/features/followers/Follower.kt
package com.possible.demo.features.followers

import timber.log.Timber

data class Follower(
        val login: String,
        val avatarUrl: String,
        val toReposFragmentCb: (login: String) -> Unit
) {
    fun onClick() {
        Timber.i("onClick: $login, $avatarUrl")

        toReposFragmentCb(login)
    }
}